whois-api/src/main/java/net/ripe/db/whois/api/httpserver/dos/AbstractDoSFilterHolder.java

```diff
@@ -0,0 +1,41 @@
+package net.ripe.db.whois.api.httpserver.dos;
+
+import jakarta.servlet.Filter;
+import jakarta.servlet.FilterChain;
+import jakarta.servlet.ServletException;
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
+import org.eclipse.jetty.servlet.FilterHolder;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+
+public class AbstractDoSFilterHolder extends FilterHolder {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(AbstractDoSFilterHolder.class);
+
+    AbstractDoSFilterHolder(final boolean dosFilterEnabled, final String trustedIpRanges){
+        if (!dosFilterEnabled) {
+            LOGGER.info("DoSFilter is *not* enabled");
+            this.setFilter(generateBasicFilter());
+            return;
+        }
+
+        this.setInitParameter("enabled", "true");
```

Comment:
This should use dosFilterEnabled , instead of setting manually to true? 
If it not enabled dos filer wont be used

Reply:
if it is not enabled we will not reach that line, we early return in line 22.
So the only possible value here is "true" I prefer put directly true rather than the property. It is more clear

Reply:
I will change it following your next advice

Reply:
Done